xwiki-platform-distribution/xwiki-platform-distribution-flavor/xwiki-platform-distribution-flavor-test/xwiki-platform-distribution-flavor-test-webstandards/src/test/it/org/xwiki/test/webstandards/CustomDutchWebGuidelinesValidator.java

```diff
@@ -83,7 +97,7 @@ private boolean isPage(String space, String page)
     public void validateRpd3s4()
     {
         if (!isPage("XWiki", "XWikiSyntax") && !isPage("XWiki", "XWikiSyntaxParagraphs")
-            && !isPage("XWiki", "XWikiSyntaxGeneralRemarks")) {
+            && !isPage("XWiki", "XWikiSyntaxGeneralRemarks") && !isTranslationDocument()) {
```

Comment:
Could you explain with a comment why we need to exclude translation documents? Thx

Reply:
Is it because we think that translation documents are technical documents and don't need to be accessible? If so we might as well exclude technical documents, no? But I'm not sure it's right to exclude translations or technical documets from being accessible?

Reply:
It's because translations documents are using the plain syntax: the linebreaks are then transformed to `<br>`. 
From what I understood in the issue comments, it's expected.
So as they are not meant to be accessed by the users, I thought it would be safe to only exclude them.

Reply:
> So as they are not meant to be accessed by the users, I thought it would be safe to only exclude them.

I don't know where this was discussed. So you're saying that we don't want people with accessibility issues to be able to develop on XWiki (i.e. add translations, modify them). It's a pretty important statement to me and I would prefer that we VOTE on this so that we all agree. WDYT?

Reply:
> I don't know where this was discussed

In the comments of the linked issue. The first proposal was to completely remove this validation since its purpose is not really clear. To quote @evalica: 
> I've run the HTML content through ckeditor-4/accessibility-checker/ and wave.webaim.org and I got no error for these consecutive <br> elements. Also checked the WCAG 2.1 (https://www.w3.org/TR/WCAG/) and didn't found anything specific about this.

> So you're saying that we don't want people with accessibility issues to be able to develop on XWiki (i.e. add translations, modify them).

So basically, no I'm not saying that. I'm saying this specific rule is failing on all our translations pages and is not necessary pertinent even for people with disabilities. 

> I would prefer that we VOTE on this so that we all agree

Agreed on that :)

Reply:
If we don't believe that this test is good for WCAG (and I'm not saying I agree, I'd need to think about it/discuss it) then it should be removed, and not put an exception for translations.

Reply:
I believe we should keep this test as it forces us to use good practices (i.e. not using consecutive `<br>` elements). In this case, the HTML is *generated* from translations documents and that's why maybe we should make an exception here.

Reply:
An alternative would be to introduce some dedicate parser syntax for translations to generate something nicer when viewing the page instead of the current plain/1.0. We never did it because it's totally useless from translation point of view but it's not very hard to do either.